tftp/backend.py

```diff
@@ -155,7 +154,7 @@ def read(self, size):
 
         """
         if self.state in ('eof', 'finished'):
-            return ''
+            return b''
```

Comment:
Documentation of affected methods and interfaces has to be updated so that it reflects the fact, that we return `bytes` instead of `str`. Its still, `str` in python 2.x, though, but I think, that using "bytes" terminology will not hurt